.github/buildomat/jobs/tuf-repo.sh

```diff
@@ -168,6 +183,38 @@ caboose_util_rot() {
 }
 
 SERIES_LIST=()
+
+# Create an initial `manifest-rot-all.toml` containing the SP images for all
+# boards. While we still need to build multiple TUF repos,
+# `add_hubris_artifacts` below will append RoT images to this manifest (in
+# addition to the single-RoT manifest it creates).
+prep_rot_all_series() {
```

Comment:
@iliana You probably care most about the changes here. They feel pretty janky, but (a) it wasn't clear how to fit this into the existing `add_hubris_artifacts` and (b) once we don't need the separate staging/dev and prod/rel TUF repos, we can probably drop most of what makes it ugly. Happy to do any cleanup if there's a nicer way to do this for now, though!

Reply:
No more janky than the rest of the script. :) I am excited to clean this up after all known racks have updated beyond this PR!